hack/lib/etcd.sh

```diff
@@ -118,14 +141,17 @@ kube::etcd::install() {
       ln -fns "etcd-v${ETCD_VERSION}-darwin-amd64" etcd
       rm "${download_file}"
     else
-      url="https://github.com/coreos/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz"
-      download_file="etcd-v${ETCD_VERSION}-linux-amd64.tar.gz"
+      url="https://github.com/coreos/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-${arch_linux}.tar.gz"
+      download_file="etcd-v${ETCD_VERSION}-linux-${arch_linux}.tar.gz"
       kube::util::download_file "${url}" "${download_file}"
       tar xzf "${download_file}"
-      ln -fns "etcd-v${ETCD_VERSION}-linux-amd64" etcd
+      ln -fns "etcd-v${ETCD_VERSION}-linux-${arch_linux}" etcd
       rm "${download_file}"
     fi
     kube::log::info "etcd v${ETCD_VERSION} installed. To use:"
     kube::log::info "export PATH=$(pwd)/etcd:\${PATH}"
+    if [[ ${arch_linux} != "amd64" ]]; then
+      kube::log::info "export ETCD_UNSUPPORTED_ARCH=${arch_linux}"
```

Comment:
Looks like this is coming from https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/supported-platform.md#32-bit-and-other-unsupported-systems Right?

Reply:
Can you please add a note for future?

Reply:
Done